scss/sections/_informations.scss

```diff
@@ -0,0 +1,79 @@
+#dna_info {
+  display: flex;
+  width: 80%;
+  margin: 0 10%;
+
+  .info_text {
+    width: 50%;
+
+    h1 {
+      font-size: 3rem;
+    }
+
+    p {
+      width: 70%;
+      margin: 0 0 5%;
+      font-size: 20px;
+    }
+  }
+
+  .movie {
+    width: 50%;
+
+    video {
+      width: 100%;
+      border-radius: 10px 10px 0 0;
+      margin-top: 3rem;
+    }
+  }
+}
+
+@media (max-width: 767px) {
+  #dna_info {
+    display: flex;
+    width: 80%;
+    flex-wrap: wrap;
+    flex-direction: column;
+
+    .info_text {
+      width: 100%;
+      text-align: center;
+      align-items: center;
+
+      p {
+        margin: 0 15% 5%;
+        text-align: center;
+        align-items: center;
+      }
+    }
+
+    .movie {
```

Comment:
You should use col-sm etc to steer width of element. 
instead declare it here.... :)